### ISSUE TITLE
Close button of mega menu misses  label for better accessibility

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#593]
Label for better accessibility is missing on close button of mega menu
![image](https://user-images.githubusercontent.com/7510842/32664839-bc11ac26-c632-11e7-804c-3e1ee0e419a4.png)

Label for closing mega menu should be 
German: Schliessen
French: Fermer
English: Close

**Comment by bit-pro-iew-eui** [comment]
Label Ok, NVDA reads "close"
